src/lib/mdns/Helper.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <core/CHIPError.h>
+
+#include <cstddef>
+#include <cstdint>
+
+namespace chip {
+namespace Mdns {
+
+CHIP_ERROR MakeServiceName(char * buffer, size_t bufferLen, uint64_t fabricId, uint64_t nodeId);
```

Comment:
Q:The spec refers to this name as the "instance name". ServiceName makes sense because it's for the SRV record, but I'm wondering if MakeInstanceName would make this less confusing for folks attempting to match the spec / SDK.

Reply:
Yes, you're right.

Reply:
Done